wire-ios/Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme

```diff
@@ -261,6 +261,11 @@
             value = ""
             isEnabled = "NO">
          </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "LIBDISPATCH_COOPERATIVE_POOL_STRICT"
+            value = "1"
+            isEnabled = "YES">
+         </EnvironmentVariable>
```

Comment:
This should be reverted or is useful to always have it on?

Reply:
No, I oversaw this.
Do you mind keeping it there but disabled?

Reply:
I don't mind